Jenkinsfile

```diff
@@ -333,6 +344,7 @@ pipeline {
           }
           steps {
             withCredentials([file(credentialsId: 'Azure_Secrets_File', variable: 'Azure_Secrets_File')]) {
+              build job: 'tool-turn-on-slaves', parameters: [string(name: 'RoleNameAndRGname', value: 'azure-slave-1@kernel_pipeline')], wait: false
```

Comment:
This change should be made in the commit with the commit message: Start azure test slaves dynamically when required, and not in the other commit.